CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ rosidl_generate_interfaces(
   "msg/LEDAnimation.msg"
   "msg/LEDAnimationQueue.msg"
   "msg/LEDImageAnimation.msg"
+  "msg/MotorController.msg"
```

Comment:
IMO `MotorController` name is misleading. Did you consider sth like `MotorControllerStateNamed` or `MotorControllerStateLabeled`? A similar name could precisely point the sense of the message.